lib/csvlint/validate.rb

```diff
@@ -552,7 +553,9 @@ def date_format?(klass, value, format)
     end
 
     def line_limit_reached?
-      @limit_lines.present? && @current_line > @limit_lines
+      unless @limit_lines.nil?
```

Comment:
I think the logic here is slightly changed - previously, if @limit_lines is `nil`, it will return `false`, but with the new code in that case it will return `nil`. I think that won't break anything as such, but it might be better to be explicit.

Perhaps:

```suggestion
      return false if @limit_lines.nil?
```